src/widgets/authorization/ui/RegisterForm.tsx

```diff
@@ -62,11 +64,11 @@ export const RegisterForm = () => {
   };
 
   return (
-    <Form onSubmit={handleSubmit(onSubmit)} title="Sign Up">
+    <Form onSubmit={handleSubmit(onSubmit)} title={t('register')}>
       <TextField
         fullWidth
         id="email"
-        label="Email"
+        label="E-mail"
```

Comment:
```suggestion
        label="E-mail"
```
Хоть емейл везде одинаковый, но если уже есть файл переводов, то лучше держать все в одном месте

Reply:
Исправил